adding resource management documentation part 2 (#24501)
opf/openproject: docs/system-admin-guide/users-permissions/user-attributes/README.md

# User attributes

> **User attributes** in OpenProject allow administrators to extend user profiles with additional information, such as job titles, skills, certifications or spoken languages. These attributes can be displayed on user profiles and on user cards in the [Resource management module](../../../user-guide/resource-management).

Reviewer: The data can also be shown in the user hover card.

> **User attributes** in OpenProject allow administrators to extend user profiles with additional information, such as job titles, skills, certifications or spoken languages. These attributes can be displayed on user profiles and on user cards in the [Resource management module](../../../user-guide/resource-management).

Author: ```suggestion
**User attributes** in OpenProject allow administrators to extend user profiles with additional information, such as job titles, skills, certifications or spoken languages. These attributes can be displayed on user profiles, hover cards and on user cards in the [Resource management module](../../../user-guide/resource-management).
```

## View user attributes

> To view all existing user attributes, navigate to **Administration settings → Users and permissions→ User attributes**.

Reviewer: ```suggestion
To view all existing user attributes, navigate to **Administration settings → Users and permissions → User attributes**.
```

![Create a new user attribute in OpenProject administration](open_project_system_admin_guide_project_attributes_add_button.png)

The following example shows a user attribute with the **List** format.

![Create a new user attribute form in OpenProject administration](open_project_system_guide_project_attributes_new_attribute.png)

- **Name**: The name displayed for the user attribute on user profiles and wherever the attribute is shown. 
- **Section**: If sections have been created, you can choose which section the user attribute belongs to. See [Sections](#sections) for more information. 
- **Allow multi-select**: Allows the user to assign multiple values to this custom field. 
- **Required**: Checking this enables this user attribute and makes it required for all users. It cannot be deactivated at a user level. Existing users will not require a value when being updated.

  > [!IMPORTANT]
  > User attribute of type **Boolean** can **NOT** be set to be required. 

- **Admin-only**: If you enable this, the user attribute will only be visible to administrators. All other users will not see it.
- **Editable**: Allows users to edit the attribute themselves.

> - **Show on user card**: Displays the attribute on user information cards throughout the application, including the Resource management module.

Reviewer: That's not correct. This setting is only for the user hover card that is shown, when a user name is hovered. 

On the user card in resource management you have to manually select what attributes are shown.

> - **Show on user card**: Displays the attribute on user information cards throughout the application, including the Resource management module.

Author: ```suggestion
- **Show on user card**: Displays the attribute on user hover cards (information shown when you hover over a user name).
```
